src/swagger/auth.swagger.js

```diff
@@ -104,3 +104,26 @@
  *       '401':
  *         description: Incorrect credentials.
  */
+/**
+ * @swagger
+ * /api/auth/logout:
+ *   get:
+ *     tags:
+ *       - User
+ *     name: Login
+ *     summary: Log a user in a system
```

Comment:
Does this route really log a user in the system?

Reply:
okay, corrected